src/openrct2/entity/Peep.cpp

```diff
@@ -1012,9 +1012,19 @@ void peep_problem_warnings_update()
              disgust_counter = 0, toilet_counter = 0, vandalism_counter = 0;
     uint8_t* warning_throttle = gPeepWarningThrottle;
 
+    int32_t inQueueCounter = 0;
+    int32_t tooLongQueueCounter = 0;
+    std::map<RideId, int32_t> queueComplainingGuestsMap;
```

Comment:
Variables above are in `snake_case`, while this isn't. Might be worth changing to make it consistent?